src/presentation/api/AdminController.ts

```diff
@@ -0,0 +1,29 @@
+import { Response } from 'express';
+import { Service } from 'typedi';
+
+import AsyncRouter from '../../api/AsyncRouter';
+import { isAuthenticated } from '../../api/middleware/isAuthenticated';
+import { AuthenticatedRequest } from '../../api/AuthenticatedRequest';
+
+import { BulkCreateUsers } from '../../application/service/users/BulkCreateUsers';
+import { CreateUserCommand } from '../commands/users';
+import { UserTransformer } from '../transformers/users';
+
+@Service()
+export class AdminController {
+  constructor(private bulkCreateUsers: BulkCreateUsers, private userTransformer: UserTransformer) {}
+
+  public routes() {
+    const route = new AsyncRouter();
+
+    route.get('/users', isAuthenticated, async (req: AuthenticatedRequest, res: Response) => {
```

Comment:
Should we have a permission check for a new permission, e.g. `BULK_CREATE_USERS`?

Reply:
Or is it configured elsewhere?

Reply:
Yep, I haven't added the permissions yet